lmdeploy/lite/apis/kv_qparams.py

```diff
@@ -124,40 +123,16 @@ def main(model: str,
     k_obs_list = list()
     v_obs_list = list()
 
-    if offload:
-        import warnings
-        warnings.warn('You are using the `offload` mode, in which the '
-                      'modules in the `OFFLOAD_MOD_MAP` will be moved to '
-                      'the GPU during forward and kept on the CPU at other '
-                      'times to save GPU memory.')
-        if type(model) not in OFFLOAD_MOD_MAP:
-
-            warnings.warn(f'{type(model)} is not in the `OFFLOAD_MOD_MAP`,'
-                          f'and by default, offloading will be done on '
-                          '`nn.Linear`. You can add more robust modules to '
-                          'the `OFFLOAD_MOD_MAP` for faster speed.')
-            offload_mod = OFFLOAD_MOD_MAP[type(model)]
-        with memory_efficient_inference(model, offload_mod):
-            for data in tqdm(calib_loader, desc='Calibrating: '):
-                if isinstance(data, torch.Tensor):
-                    output = model(data.to('cuda'))
-                else:
-                    output = model(data[0].to('cuda'))
-                kv_cache = output.past_key_values
```

Comment:
use the `KV_CACHE_SIGNATURE` of `lmdeploy/lite/defaults.py`?

Reply:
This is just a temporary solution, and this script will also be deprecated. The quantization parameters of kv cache can be obtained through the `calibrate.py`.